commands/gamble.js

```diff
@@ -0,0 +1,202 @@
+const _ = require('lodash');
+
+const gamble = {
```

Comment:
es modules?

Reply:
Still not in native Node... unless added in 8.5? Even then, we'd have to use the `.mjs` extension. 🤢

Reply:
why not use mjs?

Reply:
The "Still not in native Node" thing is the bigger issue.